examples/simple/README.md

```diff
@@ -4,8 +4,12 @@
 
 ### Install
 
+Start fetchr root:
+
 ```js
 npm install
+cd examples/simple
+npm install
```

Comment:
The app.js references libs/fetcher.client.js in the root which requires lodash which is not a dependency in the examples.